client/src/components/Flash/flash.css

```
@@ -1,7 +1,7 @@
 .flash-message {
```

The padding aren't applied because of specificity, adding `div` should solve this
```suggestion
div.flash-message {
```